frontend/src/static/js/v_summary.js

```diff
@@ -249,6 +268,14 @@ window.vSummary = {
       window.decodeHash();
     },
 
+    restoreMergedGroups() {
+      if (this.customMergedGroups) {
+        this.customMergedGroups.split(',').forEach((x) => {
```

Comment:
could u use window.HASH_FILETYPE_DELIMITER as the delimiter? can refactor the name to window.HASH_DELIMITER